pkg/skaffold/errors/errors.go

```diff
@@ -36,16 +37,16 @@ const (
 	FileSync    = Phase("FileSync")
 	DevInit     = Phase("DevInit")
 	Cleanup     = Phase("Cleanup")
-
-	// Report issue text
-	reportIssueText = "If above error is unexpected, please open an issue https://github.com/GoogleContainerTools/skaffold/issues/new to report this error"
 )
 
 var (
-	setOptionsOnce sync.Once
-	skaffoldOpts   config.SkaffoldOptions
+	// Report issue text
+	reportIssueText = fmt.Sprintf("If above error is unexpected, please open an issue %s to report this error", constants.GithubIssueLink)
```

Comment:
should we just move this to const as well?

Reply:
I usually prefer `fmt.Sprintf` over concatenation but i can change it.

Reply:
oh my bad, I didn't realize you can't use it to create const values. Up to you if you want to leave it or not